src/ai/backend/agent/docker/kernel.py

```diff
@@ -44,6 +44,8 @@
 DEFAULT_CHUNK_SIZE: Final = 256 * 1024  # 256 KiB
 DEFAULT_INFLIGHT_CHUNKS: Final = 8
 
+KERNEL_HOST = "127.0.0.1"
```

Comment:
Is there a reason to declare the kernel host as global?

Reply:
It’s probably because the other global variables are marked as private. How about making all of them private in this PR as well?